doc/nrf/app_dev/device_guides/thingy91/index.rst

```diff
@@ -38,5 +38,7 @@ See the subpages for information on the various aspects of Thingy:91.
 
    thingy91_features
    thingy91_connecting
+   thingy91_updating_fw_celmon
+   thingy91_cloud_connecting
```

Comment:
```suggestion
   thingy91_cloud_connecting
   thingy91_updating_fw_celmon
```
Could be interchanged as the same order is followed for nRF91 DKs. Cloud connectivity then firmware updates.